public/images/dashboard/apply-white-bg-icon.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="72" height="72" viewBox="0 0 72 72" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Try naming the icons similar names for readability
ie
apply-for-benefit-icon-background (for grey circle background)
apply-for-benefit-icon (for regular icons)

Reply:
hmm it seemed implicit to me.. but I'm lazy and think differently; updated.